### PR TITLE
Command Migration] Make the "drink beer" into a proper command

### PR DESCRIPTION
**Pull request by Fayyne**
Let me know if anything seems off, I dug around other commands, and look around in what I was calling to make sure messages would still prop. as needed, and I think I got it, but still learning the process flow of the app.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,8 @@
 /Initium-Odp/war/WEB-INF/lib/appengine-endpoints.jar
 /Initium-Odp/war/WEB-INF/lib/appengine-endpoints-deps.jar
 /Initium-Odp/war/WEB-INF/lib/appengine-jsr107cache-1.9.36.jar
+*.iml
+*.eml
+.idea/
+.classpath
```
Comment (Emperorlou) [inline .gitignore]: I don't want to ignore the class path so new devs have something to work with, even if it's wrong.

File: Initium-Odp/war/odp/javascript/script.js [hunk 2]
```diff
@@ -1547,11 +1547,12 @@ function deleteAndRecreateCharacter(currentCharName)
 	});
 }
 
-function doDrinkBeer()
+function doDrinkBeer(eventObject)
 {
 	confirmPopup("Drink Beer", "Are you sure you want to drink? It might affect your combat abilities...", function(){
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/script.js]: Maybe we shouldn't warm them... For fun ^^

File: .gitignore (hunk 1 above)
Comment (Fayyne) [inline .gitignore]: FUDGE didn't mean to put that one in there, habit >.>

File: Initium-Odp/war/odp/javascript/script.js [hunk 3]
```diff
@@ -1547,11 +1547,12 @@ function deleteAndRecreateCharacter(currentCharName)
...
-		window.location.href = "ServletCharacterControl?type=drinkBeer"+"&v="+window.verifyCode;
-		
+        closeAllPopups();
```
Comment (Emperorlou) [inline Initium-Odp/war/odp/javascript/script.js]: Let's not close pop-ups or tooltips anymore too

**Pull request merged**